tests/js/HappinessLintBearTest.py

```diff
@@ -17,3 +23,18 @@
 HappinessLintBearTest = verify_local_bear(HappinessLintBear,
                                           valid_files=(good_file,),
                                           invalid_files=(bad_file,))
+
+
+@generate_skip_decorator(HappinessLintBear)
+class HappinessLintBearConfigTest(LocalBearTestHelper):
+
+    VALUE_ERR_RE = ('Set "use_spaces=False" for HappinessLintBear \
```

Comment:
Do not use line continuation `\`.

Use automatic string concatenation.

```
    VALUE_ERR_RE = ('Set "use_spaces=False" for HappinessLintBear '
                    'to avoid conflicts with other bears')
```

Reply:
Earlier i used this but @Makman2 asked me to wrap it inline.

Reply:
no I meant to inline it where you use `self.VALUE_ERR_RE`, that's what inlining usually means ;)